code/modules/species/station/standard/krisitik.dm

```diff
@@ -0,0 +1,70 @@
+/datum/species/krisitik
+	uid = SPECIES_ID_KRISITIK
+	id = SPECIES_ID_KRISITIK
+	name = SPECIES_KRISITIK
+	name_plural = SPECIES_KRISITIK
+	default_bodytype = BODYTYPE_KRISITIK
+
+	icobase      = 'icons/mob/species/krisitik/body.dmi'
+	deform       = 'icons/mob/species/krisitik/body.dmi' // They don't have a proper one for some reason...
+	preview_icon = 'icons/mob/species/krisitik/preview.dmi'
+	husk_icon    = 'icons/mob/species/krisitik/husk.dmi'
+
+	tail = "tail"
+	icobase_tail = 1
+
+	max_additional_languages = 3
+	name_language   = LANGUAGE_ID_SQUEAKISH
+	intrinsic_languages = LANGUAGE_ID_SQUEAKISH
+
+	vision_innate = /datum/vision/baseline/species_tier_2
+
+	color_mult = 1
+
+	blurb = {"
+	Krisitik are a race of subterranean murine people from the Perseus Arm. Already significantly advanced
+	even before developing FTL, the Krisitik have exploded onto the galaxy using their mastery of nuclear technology
+	to bridge the gap between them and the galactic superpowers. The crowding in the massive underground\
+	cities of their toxic homeworld of Murith has led many Krisitik to emigrate to their colonies or towards the
+	corporate frontier. As their isolationist government becomes more open, this emigration is only increasing.
+
+	The Krisitik are highly varied people owing to their unstable genetics, which can result in lethal defects
+	if left untreated. They are a competitive, survivalist, and paranoid people known to stretch, bend, and
+	break the rules if it means being prepared for disasters to come, even if they are unlikely."}
+
+
+	// wikilink = ""
+	catalogue_data = list(/datum/category_item/catalogue/fauna/krisitik)
+
+
+	species_spawn_flags = SPECIES_SPAWN_CHARACTER
+	species_appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
+
+	flesh_color = "#AFA59E"
+	base_color  = "B2B2B2"
+
+	slowdown  = -0.5
+
+	brute_mod = 1.1
+	burn_mod  = 1.1
+	flash_mod = 1.5 //Cave creatures don't like bright lights
+	radiation_mod = 0.75 //Hardened Genes
+
+	gluttonous    = 1

```

Comment:
I am not sure if this is the var you are intending to use here:
`/// Can eat some mobs. 1 for mice, 2 for monkeys, 3 for people.
var/gluttonous`
if you want to make the hungry faster or slower
use hunger_factor (0.05 is baseline, 0.2 is tesh, 0.04 is slow metabolism)

Reply:
Krisitik would probably have no issue eating mice to be honest, even if its not considered 'polite'. I am keeping this.

Reply:
I just wanted to point out that there is a different variable you might have been aiming at